packages/transactions/src/__tests__/PaymentV2.spec.ts

```diff
@@ -120,6 +122,6 @@ describe('sign', () => {
     const signedPayment = await reserializedTxn.sign({ payer: bob })
     const decoded = proto.helium.blockchain_txn.decode(Buffer.from(signedPayment.toString(), 'base64'))
     const base64Signature = (decoded.paymentV2?.signature as Buffer).toString('base64')
-    expect(base64Signature).toEqual('TKUrksvBBTjgjoPEPhFc79+jPzaVN7VBWgKo4+GvDYZ4BO9HujpEuskMSvzVWq0T/tf2KAKd6d+fT3mYnMRYCA==')
+    expect(base64Signature).toEqual('PiDvVpuSI0PViJi/ItQnqyMPbHFh3mmZSMpBzHuJ6TXlig1hDtdUE8is9FJBbhdJbSz9jCpHHBB2O2d0pYPSCw==')
```

Comment:
No they should be ok, this hash only changed because I added the tokenType to it. You can see there are a couple tests in `integration_tests/tests/create_and_submit_payment.spec.ts` that still work without tokenType. In the app we will just have to assume any payment without tokentype is using HNT, but I didnt want to add that into the transaction because then they would fail to verify.